### ISSUE TITLE
if i clone the repo i can use it free ?

### ISSUE THREAD

**Issue by xrokz** [typpo/textbelt#154]

**Comment by daluu** [comment]
Please elaborate on what you mean "use it free"

**Comment by xrokz** [comment]
like to use it completely free, no funds

‫في الجمعة، 4 سبتمبر 2020 في 12:28 ص تمت كتابة ما يلي بواسطة ‪David Luu‬‏ <‪
notifications@github.com‬‏>:‬

**Comment by daluu** [comment]
Yes, it is free to host & use yourself. However, you'll need your own domain name & servers to host the service to call your own copy of the API rather than the public one that requires payment.

Give it a try and see if things work for you.

**Comment by Rubinskiy** [comment]
I think he means if he'll be able to send SMS for free by deploying this repo to his server.
